{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -22,38 +22,37 @@ warn_unused_ignores = true
 name = "{{ cookiecutter.project_slug }}"
 version = "{{ cookiecutter.project_version }}"
 description = "{{ cookiecutter.project_description }}"
-readme = "README.md"
+readme = [
+    "README.md",
+    "USAGE.md",
+]
 authors = [
     "{{ cookiecutter.author_full_name }} <{{ cookiecutter.author_email }}>",
 ]
 license = "{{ cookiecutter.project_license }}"
+classifiers = [
```

Comment:
Might want to add a link to https://pypi.org/pypi?%3Aaction=list_classifiers here.